pkg/appfile/appfile.go

```diff
@@ -114,7 +114,7 @@ func (wl *Workload) EvalHealth(ctx process.Context, client client.Client, access
 	if wl.SkipApplyWorkload {
 		return true, nil
 	}
-	return wl.engine.HealthCheck(ctx, client, accessor, wl.FullTemplate.Health)
```

Comment:
Please add some tests to cover this new feature.

Reply:
added